docs/fastdds/dds_layer/topic/contentFilteredTopic/writerFiltering.rst

```diff
@@ -0,0 +1,53 @@
+.. include:: ../../../../03-exports/aliases.include
+.. include:: ../../../../03-exports/aliases-api.include
+
+.. _dds_layer_topic_contentFilteredTopic_writer_side:
+
+Where is filtering applied: writer vs reader side
+=================================================
+
+Content filters may be evaluated on either side, as the DataWriter obtains the filter expression from the
```

Comment:
I'd add a cross-reference to the `ContentFilterTopic` section here. Also, I'd refer this section from there.